.github/workflows/ci.yml

```diff
@@ -202,6 +211,9 @@ jobs:
           go: false
           # setting up maven often times out on macOS
           maven: ${{ runner.os != 'macOS' }}
+          secret_vault_secretId: ${{ secrets.VAULT_SECRET_ID }}
```

Comment:
:x: Similar to the go tests, this is running on github provided runners and shouldn't use the nexus cache I think